components/listingsPage/filters.js

```diff
@@ -255,11 +263,19 @@ const Filters = ({ t }) => {
                 />
               )
             )}
-            <Heading level={4}>
+            <Heading level={4} color="#217D17">
               {t("headings.filter_by_length_of_trip")}
             </Heading>
             <Note note={note}>Please select the type of insurance first</Note>
             <SingleTrip single={single}>
+              <span style={{ fontSize: "14px" }}>
+                single trips &nbsp; &nbsp;
```

Comment:
This text looks too small to me, as well as the text on line `294` for the annual trip

Reply:
Hmm, should i make it bolder. I dont want it look as big as the others